cupy/cuda/compiler.py

```diff
@@ -331,11 +341,11 @@ def _compile_with_cache_cuda(
     if _get_bool_env_variable('CUPY_CUDA_COMPILE_WITH_DEBUG', False):
         options += ('--device-debug', '--generate-line-info')
 
-    env = (arch, options, _get_nvrtc_version(), backend)
+    env = (arch, options, _get_nvrtc_version(), backend, log_stream)
```

Comment:
I'd not change this line. This is for generating a key, but `log_stream` would not affect compilation results, so it won't help here.
```suggestion
    env = (arch, options, _get_nvrtc_version(), backend)
```